Swiftification/Array.swift

```diff
@@ -53,16 +53,16 @@ public extension Array {
     mutating func shuffleInPlace() {
         guard count > 1 else { return }
         
-        for index in (1..<count).reverse() {
-            let newIndex = Int.random(0...index)
+        for index in (1..<count).reversed() {
+            let newIndex = Int.random(0..<(index + 1))
```

Comment:
I have seen this a bunch with converting.  its like they don't want to use it unless its a Range with `..<`